README.ffmpeg

```diff
@@ -7,6 +7,7 @@ https://www.ffmpeg.org/legal.html
 Building with FFmpeg support on *nix platforms
 ----------------------------------------------
 Enable FFmpeg support at cmake configure-time with `-DFFMPEG=ON`.
+When enabling FFMpeg support, if using a custom FFMpeg mingw build directory, you must also specify `-DFFmpeg_INCLUDE_DIR=` and `-DFFmpeg_LIBRARIES=` with their respective paths.
```

Comment:
Line break for 80(?) character limit. At least, it’s supposed to be 80, never counted for this file since Johan wrote it...